cmd/describe/cluster/cmd.go

```diff
@@ -102,6 +102,29 @@ func run(cmd *cobra.Command, argv []string) {
 	var upgradeState *cmv1.UpgradePolicyState
 	var controlPlaneScheduledUpgrade *cmv1.ControlPlaneUpgradePolicy
 
+	// SDN -> OVN Migration Status
+	migrationsStr := ""
+	collection, err := r.OCMClient.FetchClusterMigrations(cluster.ID())
+	if err != nil {
+		r.Reporter.Warnf("Failed to get cluster migrations: %v", err)
```

Comment:
If the reason for the failure is that the toggle isn't enabled I feel like it shouldn't be reported at all :thinking: